common/static/common/js/discussion/utils.js

```diff
@@ -444,19 +463,52 @@
             }
         };
 
-        DiscussionUtil.abbreviateHTML = function(html, minLength) {
+        DiscussionUtil.convertMath = function(element, alreadyEscaped) {
+            var elementContent = element.html();
+            if (alreadyEscaped){
+                // markdownWithHighlight requires unescaped text, so we must unescape it.
+                // safe-lint: disable=javascript-escape
+                elementContent = _.unescape(edx.HtmlUtils.HTML(elementContent));
+            }
```

Comment:
So you are saying that the following works:
```
elementContent = _.unescape(edx.HtmlUtils.HTML(elementContent));
```
But, the following does not work?:
```
elementContent = _.unescape(elementContent);
```
The call to HTML() should be a no-op, so I'm not sure why you would need it.  That's all I'm saying.